scripts/setup_gitpod.sh

```diff
@@ -2,7 +2,7 @@
 # This script does the setup needed for Gitpod
 
 # Setup pre-commit hooks
-pyenv install 3.11 --skip-existing # must match python version in .pre-commit-config.yaml
+pyenv install 3.11 --skip-existing # must match python version in .pre-commit-config.yml
```

Comment:
```suggestion
pyenv install 3.11 --skip-existing # must match python version in .pre-commit-config.yaml
```
I can see how this one is a bit counterintuitive, but although this is Docker-adjacent, it is not actually related to Docker.